vendor/github.com/docker/docker/vendor.conf

```diff
@@ -26,7 +26,7 @@ github.com/imdario/mergo v0.3.6
 golang.org/x/sync 1d60e4601c6fd243af51cc01ddf169918a5407ca
 
 # buildkit
-github.com/moby/buildkit e57eed420c7573ae44875be98fa877175b4677a1
+github.com/moby/buildkit 46f9075ab68a07df2c40ae6e240ce4f9392b3a66 git://github.com/tiborvass/buildkit.git
```

Comment:
I ignored this bump, because it's a temporary fork, and this didn't bring in changes to vendored files.